### PR TITLE
[Bug 1444044]Changed the Welcome Community Message 

### PR DESCRIPTION
**Pull request by turtlelovesshoes**
Per https://bugzilla.mozilla.org/show_bug.cgi?id=1444044#c7
It looked like it should be pretty easy to update the text with updated information for this message that is triggered when a user signs up as a contributor then makes their first contribution. 

+r

### CODE REVIEW

**Comment by vesper-mozilla** [comment]
@pmac @jpetto Hey hey, what are the next steps to get this into Kitsune?

Cheers :)

**Comment by pmac** [comment]
Hi! Sorry for failing to get this in. I'll take a look today.

File: kitsune/users/templates/users/email/contributor.ltxt [hunk 1]
```diff
@@ -36,21 +36,10 @@ most-requested but have no replies:
 
 {# L10n: This is an email.  Whitespace matters! #}
 {% trans %}
-As you get started answering questions that look easy you
-might run into a few questions. Like, what happens when an
-angry user does not want a work around or what happens next?
-The Contributor Quality Training may have these answered. It
-is possible to get started here:
+Social Support Helper - Respond to tweets about Firefoxand as the Firefox brand account:
```
Comment (pmac) [inline kitsune/users/templates/users/email/contributor.ltxt]: `Firefoxand` needs a space I think.

**Comment by pmac** [comment]
I was able to do the rebase and have merged these changes into master in a3d0e6faacbb35dd1ae0a5f034a90327811c3821. Let me know if you see any further issues. I can get this deployed to production today I think.

**Comment by pmac** [comment]
This is now in production.

**Pull request closed without merging**